_docs/_latest/use/cli/inventory.md

```diff
@@ -32,10 +34,10 @@ forseti inventory create
 
 You can track the status by using the `list` command.
 
-### Create a new inventory along with a data model
+### Create a new inventory and data model
```

Comment:
Same comment as above regarding checking links to the old heading.

Reply:
As above, redirect or persistent anchor?

Reply:
Resolved